components/markets/ScalarPriceRange.tsx

```diff
@@ -56,15 +42,36 @@ const ScalarPriceRange = ({
     }
   };
 
+  const lowerDisplay =
```

Comment:
Think there is some rounding missing somewhere

Staging:
<img width="414" alt="image" src="https://github.com/zeitgeistpm/ui/assets/4950844/dfc26da2-afd8-497e-875e-eeef5874b1ee">

This branch:
<img width="414" alt="image" src="https://github.com/zeitgeistpm/ui/assets/4950844/1f674c88-15dd-49fa-89a2-8f7392deb3c9">

Reply:
> Think there is some rounding missing somewhere
> 
> Staging: <img alt="image" width="414" src="https://user-images.githubusercontent.com/4950844/245410730-dfc26da2-afd8-497e-875e-eeef5874b1ee.png">
> 
> This branch: <img alt="image" width="414" src="https://user-images.githubusercontent.com/4950844/245410800-1f674c88-15dd-49fa-89a2-8f7392deb3c9.png">

I dont think I want to round here in case they are working with small numbers, but maybe to fix to 2 or 3 decimals?